packages/editor-ui/src/components/mixins/pinData.ts

```diff
@@ -0,0 +1,18 @@
+import Vue from 'vue';
+import { INodeUi } from "@/Interface";
+import { IDataObject } from "n8n-workflow";
+
+export const pinData = Vue.extend({
+	props: ['name'],
+	computed: {
+		node (): INodeUi | undefined {
+			return this.$store.getters.nodesByName[this.name] as INodeUi | undefined;
+		},
+		pinData (): IDataObject {
+			return !!this.node && this.$store.getters['pinDataByNodeName'](this.node.name);
+		},
```

Comment:
```suggestion
		pinData (): IDataObject {
			return this.node !== null && this.$store.getters['pinDataByNodeName'](this.node.name);
		},
```

Reply:
In some places it's typed as undefined, in some places it's typed as null. Double negation will cover both without typescript complaining. 😅